src/Microsoft.AspNetCore.Mvc.Razor.Extensions/Microsoft.AspNetCore.Mvc.Razor.Extensions.csproj

```diff
@@ -11,6 +11,16 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="..\Microsoft.AspNetCore.Razor.Language\BoundAttributeDescriptorComparer.cs">
+      <Link>BoundAttributeDescriptorComparer.cs</Link>
+    </Compile>
+  </ItemGroup>
```

Comment:
One other solution I can think of is to add IVT between `Razor.Language` and `Razor.Extensions`. I assume this is okay because we already do that with `CodeAnalysis.Razor`, `LanguageServices.Razor` etc.